frontend/src/api/hello.ts

```diff
@@ -0,0 +1,4 @@
+const fetchHello = (): Promise<string> => fetch('/api/v1/hello')
+  .then((response) => response.text());
```

Comment:
The `fetch('/api/v1/hello')` returns a `SampleObject`, so `response` should be turned into `response.json()` (not `response.text()`. This also means the `Promise` should hold a different type, other than a **string**.

(Some similar changes will need to be made on the frontend file, `App.tsx`, as well.)

Reply:
Thanks for catching that! This should be fixed now.